<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVDBZoneControlVentilationNaturalEditWidget</class>
 <widget class="QWidget" name="SVDBZoneControlVentilationNaturalEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayoutMaster">
   <item row="2" column="0">
    <widget class="QLabel" name="labelTemperatureAirRoomMaximum">
     <property name="text">
      <string>Room air temperature maximum:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditTemperatureAirRoomMaximum">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="labelConvectiveFactorUnit">
     <property name="text">
      <string>C</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="labelConvectiveFactorUnit_11">
     <property name="text">
      <string>m/s</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="labelConvectiveFactorUnit_2">
     <property name="text">
      <string>C</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelTemperatureAirRoomMinimum">
     <property name="text">
      <string>Room air temperature minimum:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditTemperatureAirRoomMinimum">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditWindSpeedMax">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditMaximumAirChangeRateComfort">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelWindSpeedMax">
     <property name="text">
      <string>Wind speed maximum:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelDisplayName">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QtExt::LanguageStringEditWidget1" name="lineEditName"/>
     </item>
     <item>
      <widget class="QtExt::ColorButton" name="pushButtonColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>3</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelMaximumAirChangeRateComfort">
     <property name="text">
      <string>Maximum air change rate comfort:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="labelConvectiveFactorUnit_12">
     <property name="text">
      <string>1/h</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtExt::ValidatingLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_ValidatingLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QtExt::ColorButton</class>
   <extends>QPushButton</extends>
   <header location="global">QtExt_ColorButton.h</header>
  </customwidget>
  <customwidget>
   <class>QtExt::LanguageStringEditWidget1</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_LanguageStringEditWidget1.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
